src/components/__tests__/header.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Header from '../header';
+
+let wrapper;
+
+beforeAll(() => {
```

Comment:
Kindly move this hook into the describe block. This ensures it's scoped to the instance of the test at all times.

Reply:
Okay. resolved.